calculator/lexer_test.py

```diff
@@ -5,20 +5,25 @@
 
 
 def test_basecase():
-    # assert lex('1 + 1') == [ 1, '+', 1 ]
-    pass
+    assert lex("1 + 1") == [1, "+", 1]
 
 
 def test_negative_weirdness():
-    # assert lex('1 - -1') == [ 1, '-', -1 ]
-    pass
+    assert lex("1 - -1") == [1, "-", -1]
 
 
 def test_no_space():
-    # assert lex('-1.2*2') == [ -1.2, '*', 2 ]
-    pass
+    assert lex("-1.2*2") == [-1.2, "*", 2]
+
+
+def test_multiple_spaces():
+    assert lex("1  + 1") == [1, "+", 1]
 
 
 def test_parens():
-    # assert lex('(2 + 3) - 3.2') == [ '(', 2, '+', 3, ')', '-', 3.2 ]
-    pass
+    assert lex("(2 + 3) - 3.2") == ["(", 2, "+", 3, ")", "-", 3.2]
+
+
+def test_RPN():
+    print("Lex: ".format(lex("1 1 +")))
+    assert lex("1 1 +") == [1, 1, "+"]
```

Comment:
Can you try `assert lex("1   1   +"`) == [1, 1, "+"]` please?

Reply:
👍